tests/fgcmcalTestBase.py

```diff
@@ -302,26 +296,26 @@ def _testFgcmFitCycle(self, instName, testName, cycleNumber,
         outputCollection = f'{instName}/{testName}/fit'
 
         if cycleNumber == 0:
-            inputCollections = f'{instName}/{testName}/buildstars'
+            inputCollections = [f'{instName}/{testName}/buildstars']
         else:
-            # We are reusing the outputCollection so we can't specify the input
-            inputCollections = None
+            # We need to know the correct input collections to be able to
+            # reuse the output collection with the pipetask API.
```

Comment:
I think I'm having a little trouble parsing this comment. If I didn't know about the discussion you and Jim had on slack, I'd be a bit lost. Not sure how to reword it though.

Reply:
How about:
```
            # In these tests we are running the fit cycle task multiple
            # times into the same output collection.  This code allows
            # us to find the correct chained input collections to use
            # so that we can both read from previous runs in the output
            # collection and write to a new run in the output collection.
            # Note that this behavior is handled automatically by the
            # pipetask command-line interface, but not by the python
            # API.
```